### DISCUSSION TITLE
How to associate repository with package?

### DISCUSSION THREAD

**Discussion by SoftCircuits** [community/community#25494] [Actions]
<p>I have a number of <a href="https://github.com/SoftCircuits" rel="noopener nofollow ugc">repositories on GitHub</a>, and virtually all of them have been published as packages on <a href="https://www.nuget.org/profiles/SoftCircuits" rel="noopener nofollow ugc">NuGet</a>. My packages combined are coming up on 20,000 downloads.</p>
<p>But when I look at any of my repositories, I see this:</p>
<blockquote>
<p>Packages<br>
No packages published<br>
Publish your first package</p>
</blockquote>
<p>When I click <em>Publish your first package</em>, I’m not entirely clear on what we’re doing.</p>
<p>Is there a way to associate my repositories with their existing NuGet packages?</p>

**Comment by jcansdale** [comment, marked as answer]
<p><a class="mention" href="/softcircuits">@softcircuits</a>,</p>
<blockquote>
<p>Is there a way to associate my repositories with their existing NuGet packages?</p>
</blockquote>
<p>If you add the following property to your project, you will be able to publish the package to GitHub Packages.</p>
<pre><code class="lang-auto">&lt;RepositoryUrl&gt;https://github.com/OWNER/REPOSITORY&lt;/RepositoryUrl&gt;
</code></pre>
<p>See here for more info:<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/packages/using-github-packages-with-your-projects-ecosystem/configuring-dotnet-cli-for-use-with-github-packages#publishing-a-package" target="_blank" rel="noopener nofollow ugc">docs.github.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="16" height="16">

<h3><a href="https://docs.github.com/en/packages/using-github-packages-with-your-projects-ecosystem/configuring-dotnet-cli-for-use-with-github-packages#publishing-a-package" target="_blank" rel="noopener nofollow ugc">Configuring dotnet CLI for use with GitHub Packages - GitHub Docs</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>At the moment there aren’t any automatic connections between packages published to <a href="http://nuget.org" rel="noopener nofollow ugc">nuget.org</a> and GitHub Packages.</p>
<p>GitHub Packages can be a good place to publish work-in-progress packages before the release version is published to <a href="http://nuget.org" rel="noopener nofollow ugc">nuget.org</a>.</p>
